ci/jenkins/test-vm.sh

```diff
@@ -116,7 +116,7 @@ function clean_antrea {
     kubectl delete -f ${WORKDIR}/antrea.yml --ignore-not-found=true
     # The cleanup and stats are best-effort.
     set +e
-    docker image prune -f --filter "until=1h" > /dev/null
+    docker system prune --force --all --filter until=1h > /dev/null
```

Comment:
Could you verify if running `system prune --force --all` will delete unused antrea and "none" images on testbed?

Reply:
Yes, it can delete those images.

Reply:
@XinShuYang Yes, those images can be deleted. Also, I want to confirm that VM e2e test will be run on its separate testbed. In this way, I guess we do not need  "until=1h" here since there are no other dependencies on these builds.

Reply:
Right, I tested on the testbed and it can work.